gulp/check.js

```diff
@@ -32,3 +32,12 @@ module.exports.checkDepsExact = function (options, cb) {
 		cb();
 	}
 };
+
+module.exports.lint = function (options) {
```

Comment:
https://github.com/epam/ketcher/pull/49/files#diff-2c34f4245be5d3938b03ed57dc1e45d3L53 - Artur removed gulp lint task in scope of his merged PR, verify with him that changes

Reply:
Done